app/eq.py

```diff
@@ -87,31 +98,33 @@ async def build(self):
             self._language_code = self._sample_attributes['language']
 
         self._payload = {
-            'jti': str(uuid4()),  # required by eQ for creating a new claim
-            'tx_id': self.
-            _tx_id,  # not required by eQ (will generate if does not exist)
+            'jti': str(uuid4()),
+            'tx_id': self._tx_id,
             'iat': int(time.time()),
-            'exp': int(time.time() +
-                       (5 * 60)),  # required by eQ for creating a new claim
-            'collection_exercise_sid': self._collex_id,  # required by eQ
+            'exp': int(time.time() + (5 * 60)),
+            'collection_exercise_sid': self._collex_id,
             'region_code': self.convert_region_code(self._region),
```

Comment:
although the JIRA asks for `region_code` to be hardcoded, I guess it is reasonable to leave as-is ?